solutions/swb-app/src/accountRoutes.ts

```diff
@@ -52,7 +52,7 @@ export function setUpAccountRoutes(router: Router, hostingAccountService: Hostin
     })
   );
 
-  router.patch(
+  router.put(
     '/awsAccounts/:id',
```

Comment:
hmm, shouldn't this stay as PATCH? All of our update APIs are PATCH because the intention is the user can update specific fields instead of all fields.